### ISSUE TITLE
Excuse me, is there any suitable all2all or X2X detection algorithm?

### ISSUE THREAD

**Issue by wp275006311** [wanghangpsu/MMBD#3]
I have read the detection algorithm you proposed and conducted actual verification, which shows very high performance. However, this detection algorithm can only detect the attack mode of all2one, but the detection mode of all2all or x2x is invalid. Please kindly ask if there are other simple and effective detection methods for them? thank you

**Comment by wanghangpsu** [comment]
Yes, based on the assumption. We assume there is only one target class, so that we can use the remaining classes' margin to learn a null distribution. 

However we might be able to observe the margin difference between target class and non-target classes. i.e., the margin of the target class is usually larger than the margin of non-target class
